spec.md

```diff
@@ -245,6 +243,32 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD chose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are valid:
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| error | string | A single word uniquely identifying the error condition. |
+| description | string | A meaningful error message explaining why the request failed. | 
+
+### Error Codes
+
+There are failure scenarios described throughout this specification for which
+the `error` field MUST contain a specific string. Service Broker authors MUST
+use these error codes for the specified failure scenarios. For other scenarios,
+Service Broker authors SHOULD choose a suitable one-word error code.
+
+| Error | Reason | Expected Action |
```

Comment:
The error codes are now defined twice in the spec. This may lead to inconsistencies in the future. I don't think we need them here.

Reply:
the intent is now each time they are referenced later in the spec, they refer back to this section. If we missed a spot, that is an issue with the PR and we would like to catch and fix it.